src/app/patient-creation/patient-creation.component.html

```diff
@@ -289,12 +289,34 @@ <h2>Patient Registration</h2>
               </div>
             </div>
           </div>
+          <div class="col-md-6 col-sm-6 col-lg-6 col-xs-6">
+            <div class="form-group">
+              <label for="careGivername">Caregiver Name</label>
+              <input type="text" class="form-control" [(ngModel)]="careGivername" name="careGivername"
+                placeholder="Caregiver Name" required>
+            </div>
+          </div>
+          <div class="col-md-6 col-sm-6 col-lg-6 col-xs-6">
+            <div class="form-group">
+              <label for="relationshipToCareGiver">Relationship</label>
+              <select [(ngModel)]="relationshipToCareGiver" [ngModelOptions]="{standalone: true}" class="form-control">
+                <option value="null" ></option>
+                          <option  *ngFor="let patientRelationshipType of patientRelationshipTypes" [ngValue]="patientRelationshipType.display">{{patientRelationshipType.display}}</option>
+                      </select>
+              <!-- <input type="text" class="form-control" [(ngModel)]="relationshipToCareGiver"
+                name="relationshipToCareGiver" placeholder="Relation to caregiver" required> -->
```

Comment:
```suggestion
              <!-- <input type="text" class="form-control" [(ngModel)]="relationshipToCareGiver"
                name="relationshipToCareGiver" placeholder="Relation to caregiver" required> -->
```
This can be removed if not needed